src/main/resources/templates/home.html

```diff
@@ -19,8 +22,9 @@
                     <li>And much more...</li>
                 </ul>
             </div>
-            <a href="login.html" class="button">Login</a>
-            <a href="signup.html" class="button">Sign Up</a>
+            <a th:href="@{/login}" class="button">Login</a>
```

Comment:
The redirects also needed fixed. I used thymeleaf href to tag the route parameter in the controller I created.